src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -63,6 +67,8 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 	}
 
 	processEdit(edit: DesignerEdit): void {
+		const editEvent = this.createEditActionEvent(edit);
```

Comment:
Might as well just create this down where you actually send it - no reason for it to be up here